google-cloud-dlp/acceptance/google/cloud/dlp/v2beta1/dlp_service_smoke_test.rb

```diff
@@ -17,12 +17,12 @@
 require "minitest/autorun"
 require "minitest/spec"
 
-require "google/cloud/dlp"
+require "google/cloud/dlp/v2beta1"
 
 describe "DlpServiceSmokeTest" do
   it "runs one smoke test with inspect_content" do
 
-    dlp_service_client = Google::Cloud::Dlp.new
+    dlp_service_client = Google::Cloud::Dlp::V2beta1::DlpServiceClient.new
```

Comment:
Would it be a good idea to keep the original require and call `Google::Cloud::Dlp.new version: :v2beta1` here?

Reply:
Great suggestion - done.